README.md

```diff
@@ -1,24 +1,63 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Ruby](https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white) ![Rails](https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white)
```

Comment:
Kindly refer to the [Microverse Template](https://github.com/microverseinc/readme-template) and use it. 
- It contains a table of content which should be present.
- The table of content are links which point to the appropriate section.
- Kindly use it now and in all future projects.